### PR TITLE
prevent replacing any link that is javascript code

### PR DESCRIPTION
**Pull request by a-bougouin**
### Purpose/goal of Pull Request
Prevent replacing any link that is javascript code

### Comments
Only the case of `href="javascript:void(0);"` was handle, but any javascript code should be handled.

### CODE REVIEW

File: lib/wovnrb/html_replacers/link_replacer.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def replace(dom, lang)
 
         href = node.get_attribute('href')
         next if href =~ /^\s*\{\{.+\}\}\s*$/
-        next if href =~ /javascript:void\(/
+        next if href =~ /^\s*javascript:/
```
Comment (hythof) [inline lib/wovnrb/html_replacers/link_replacer.rb]: Could you add case insensitive to regular expression?
As `"JAVASCRIPT:alert(1)" =~ /^\s*javascript:/i`.

Comment (a-bougouin) [reply lib/wovnrb/html_replacers/link_replacer.rb]: I will, thanks

Comment (a-bougouin) [reply lib/wovnrb/html_replacers/link_replacer.rb]: done, thanks

File: test/lib/html_replacers/link_replacer_test.rb [hunk 2]
```diff
@@ -42,7 +42,25 @@ def test_replace_javascript_link_query
...
+    def test_replace_javascript_code_link_query
+      replacer = LinkReplacer.new('query', get_header)
+      dom = Wovnrb.get_dom('<a href="javascript:onclick($(\'.any\').slideToggle());">link text</a>')
+      replacer.replace(dom, Lang.new('en'))
+
+      link = dom.xpath('//a')[0].get_attribute('href')
+      assert_equal('javascript:onclick($(\'.any\').slideToggle());', link)
+    end
+
+    def test_replace_upercased_javascript_code_link_query
```
Comment (suplo) [inline test/lib/html_replacers/link_replacer_test.rb]: typo `uppercase`

**Pull request merged**